articles/tutorials/create-a-sample-app/02-create-a-petclinic-csharp/create-a-petclinic-csharp.md

```diff
@@ -220,43 +255,43 @@ Once the staged `Changes` comes into view, you can review the proposed code chan
+On clicking the file, Intent Architect will start up an external "diff" tool which you can use to review the changes. By default Intent Architect will use Visual Studio Code if it can find it, otherwise you can manually configure any other "diff" tool in Settings.
 
-![Service Implementation](images/software-factory-run-staging-diff-ownerrestcontroller-impl.png)
+![A "diff" showing the staged contents of OwnerService.cs](images/software-factory-run-staging-diff-ownerrestcontroller-impl.png)
 
-For the purposes of this tutorial, don't apply the changes at this time and instead click `CANCEL`.
+Don't apply the changes for the moment and instead click `CANCEL`.
 
 Click the `Modules` option located in the left pane of the window and install the following modules:
 
 - `Intent.EntityFrameworkCore.Repositories`
 - `Intent.Application.ServiceImplementations.Conventions.CRUD`
 
-Execute the Software Factory again, click the file `OwnerService` in the changes and observe how its output has changed from the previous run..
+Execute the Software Factory again, click the file `OwnerService` in the changes and observe how its output has changed from the previous run.
 
 ![Service Implementation CRUD](images/software-factory-run-staging-diff-ownerrestcontroller-impl-crud.png)
 
 > [!NOTE]
-> Observe that the operations are now populated with code that will facilitate CRUD operations.
+> Observe that the operations are now populated with code that will facilitate [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) operations.
 >
 > The `Intent.Application.ServiceImplementations.Conventions.CRUD` module automatically checked for operations which followed simple conventions and generated basic implementations where found.
 >
 > Anything not meeting these conventions was ignored and no implementations were generated.
-
+>
 > [!WARNING]
 >
-> It is strongly recommended that one _always_ reviews all files in the `Changes` tab of the Software Factory Execution to ensure that all changes that will be applied are what you expect.
+> It is recommended to always review all files in the `Changes` tab of the Software Factory Execution to ensure that all changes that will be applied are as expected.
 
-Click `APPLY CHANGES`.
+This time we will commit the changes, so click `APPLY CHANGES`.
 
-> [!NOTE]
-> To locate the `OwnerService.cs` file that you inspected before, after the code is generated, open up your File explorer in the directory where you have created this Intent Application, locate the `PetClinicRest` folder and open the `PetClinicRest.sln` file.
-> In the IDE, expand the `PetClinicRest.Application` project then expand the `Implementation` folder and then notice where the `OwnerService.cs` file is.
```

Comment:
This seemed superfluous, below I've detailed how to find the output, so they could always find the file in their IDE, or looking in the folders.

Reply:
Hi Jon. Just be aware that this was added from the feedback from Andrew (not Nixon) that Gareth relayed. Unless you made a reference link I would suggest we still leave some indication of where they can go to find that source code.

Reply:
@garethbaars, interested in your quick opinion here.

Before it looked like this:

![image](https://user-images.githubusercontent.com/1290262/128475558-d586d626-27b5-481d-a69c-764bbe9a1676.png)

And now it looks [like this](https://intentarchitect.com/docs/articles/tutorials/create-a-sample-app/02-create-a-petclinic-csharp/create-a-petclinic-csharp.html#test-the-generated-applications-back-end):

![image](https://user-images.githubusercontent.com/1290262/128475670-66bb828a-cd9e-41f1-9d88-d797b85d0769.png)

Do you feel that concern is adequately addressed, or if we should change things a little?

Reply:
Hey @JonathanLydall,

Happy with this. Let's see what other devs experience when they go through the tutorial and incorporate their feedback.